UPGRADING.md

```diff
@@ -0,0 +1,115 @@
+# 1.0.0 Migration Guide
+
+The 1.0 release of the `google-cloud-phishing-protection` client is a significant upgrade based on a [next-gen code generator](https://github.com/googleapis/gapic-generator-python), and includes substantial interface changes. Existing code written for earlier versions of this library will likely require updates to use this version. This document describes the changes that have been made, and what you need to do to update your usage.
+
+If you experience issues or have questions, please file an [issue](https://github.com/googleapis/python-phishingprotection/issues).
+
+## Supported Python Versions
+
+> **WARNING**: Breaking change
+The 1.0.0 release requires Python 3.6+.
+
+
+## Method Calls
+
+> **WARNING**: Breaking change
+Methods expect request objects. We provide a script that will convert most common use cases.
+
+* Install the library
+
+```py
+python3 -m pip install google-cloud-phishing-protection
+```
+
+* The scripts `fixup_phishingprotection_v1beta1_keywords.py` shipped with the library. It expects
+an input directory (with the code to convert) and an empty destination directory.
+
+```sh
+$ fixup_phishingprotection_v1beta1_keywords.py --input-directory .samples/ --output-directory samples/
+```
+
+**Before:**
+```py
+from google.cloud import phishingprotection_v1beta1
+client = phishingprotection_v1beta1.PhishingProtectionServiceClient()
+parent = client.project_path('[PROJECT]')
+# TODO: Initialize `uri`:
+uri = ''
+response = client.report_phishing(parent, uri)
+```
+
+
+**After:**
+```py
+from google.cloud import phishingprotection_v1beta1
+client = phishingprotection_v1beta1.PhishingProtectionServiceClient()
+response = client.search_hashes(request={"parent": "project/project_number", "uri": "''"})
```

Comment:
I think it would be more helpful for the before and after examples to use the same method.

Reply:
Thanks! it's updated to `report_phishing`